src/docs/quick-start.rst

```diff
@@ -1,12 +1,182 @@
+FreeIPA, AD, and LDAP.
+
+Quick Start IPA
+***************
+
+Before starting, make sure you have the following information.
+
+- Administrator credentials *i.e. admin*
+- FreeIPA domain name *i.e. sssd.io*
+- FreeIPA server hostname *i.e. ipa1.sssd.io*
+- FreeIPA server IP *i.e. 1.2.3.4*
+
+Install the necessary packages, for RHEL and clones the package is named **ipa-client**, and for Fedora it’s **freeipa-client**.
+
+.. code-tabs::
+
+    .. rhel-tab::
+
+        yum install -y ipa-client
+
+    .. fedora-tab::
+
+         dnf install -y freeipa-client
+
+Make sure DNS is pointing to your IPA server(s) by checking **/etc/resolv.conf** and it having it contain an entry with the IPA server IP, if it’s missing go ahead and add it, as the first entry.
+
+.. code-block:: text
+  :caption: /etc/resolv.conf
+
+    search sssd.io
+    nameserver 1.2.3.4
+    nameserver 1.1.1.1
+
+Now the `ipa-client-install` command will work. Run the command `ipa-client-install` and follow the prompts asking for your domain and server and then a user that can join the domain, which will be the administrator user.
+
+.. code-block:: text
+
+    ipa-client-install
+
+After it’s finished, test to see if the users in IPA show up on the system, by running `getent` or `id <IPA_USER>`
+
+.. code-block:: text
+
+    id admin
+
+If you have auto mounts configured, run ipa-client-automount to enable that feature. Use the -U flag for unattended.
+
+.. code-block:: text
+
+    ipa-client-automount -U
+
+
+Quick Start AD
+**************
+
+Before starting make sure you have the following information:
+
+- Domain user credentials *i.e. Administrator*
+- AD Domain name *i.e. sssd.io*
+- AD Server IP *i.e. 1.2.3.4*
+- AD Server hostname *i.e. ad1.sssd.io*
+
+Install the necessary packages, for RHEL and clones the packages are *sssd, adcli, realm, oddjob, and oddjob-mkhomedir*
+
+.. code-block:: text
+
+    yum install -y sssd adcli realm oddjob oddjob-mkhomedir
+
+Make sure DNS is pointing to your AD server(s) by checking **/etc/resolv.conf** and it having it contain an entry with an AD server IP, if it’s missing go ahead and add it, as the first entry.
+
+.. code-block:: text
+
+    search sssd.io
+    nameserver 1.2.3.4
+    nameserver 1.1.1.1
+
+Now you can issue the *realm join* command with the domain name in order to join the domain.
+
+.. code-block:: text
+
+    realm join sssd.io
+
+It will default and use the *Administrator* user, add the -u flag to specify a different user account to join the domain.
+
+.. code-block:: text
+
+    realm join -u jsmith sssd.io
+
+Now see if it works, and issue an id command.
+
+.. code-block:: text
+
+    id administrator@sssd.io
+
+If you want to use short names, edit **sssd.conf** and set **use_fully_qualified_names** to **false**.
+
+.. note::
+
+    To avoid conflicts when using short names in a multidomain environment, configure SSSD using :doc:`short names <../design-pages/pages/shortnames>`.
```

Comment:
I don't understand this sentence.

Reply:
I'm thinking about removing it, but I did rephrase it, for it to make more sense. 

~     In the event of user name conflict, jsmith@sssd.io, jsmith@child.sssd.io for example. You can configure a resolution order using so one user will be prioritized:doc:`short names <../design  -pages/pages/shortnames>`.

I'm leaning towards removing it more now, opinion?